amplify/team-provider-info.json

```diff
@@ -20,6 +20,16 @@
       },
       "auth": {
         "newdawnfallinb4b2fda2": {}
+      },
+      "storage": {
```

Comment:
new-dawn-users is not a good name. Instead, we should call it new-dawn-sponsors and then we should also have another bucket called new-dawn-veterans

It should follow this design:

s3 Bucket
// unique sponsors directory get create on user creation
/sponsor
     /unique sponsor
// after sponsor creates veteran profile
/veteran
   /unique veteran1 the sponsor creates
   /unique veteran2 the sponsor creates
   /unique veteran3 the sponsor creates
   /unique veteran4 the sponsor creates
   /unique veteran5